src/components/DetailsList/DetailsList.Props.ts

```diff
@@ -134,6 +134,9 @@ export interface IDetailsListProps extends React.Props<DetailsList> {
 
   /** Boolean value to indicate if the role application should be applied on details list. Set to false by default */
   shouldApplyApplicationRole?: boolean;
+
+  /** Callback for selection zone to indicate whether the item was selected */
+  onItemSelected?: (isSelected: boolean) => void;
```

Comment:
The main idea behind that was there was no indication is selection zone whether the item was selected using spacebar or click. I wanted to know that because we want to add an aria alert when the list item gets selected.